frontend/src/components/signup.tsx

```diff
@@ -27,22 +37,28 @@ export const SignUp = () => {
                     email,
                     password, 
                 })
-        
+
```

Comment:
nit: generally with chained promises you do want to have them together and not separate with the new line